test/commands/test-cli-std.js

```diff
@@ -14,7 +17,8 @@ module.exports = {
   },
   planifyInputs: (inputs, stdin) => {
     for (let i = 0; i < inputs.length; i += 1) {
-      setTimeout(() => stdin.send(`${inputs[i]}\n`), 500 + i * 100);
+      // WARNING: Smelly code..., could break if the test pre-loading time increases.
+      setTimeout(() => stdin.send(`${inputs[i]}\n`), TIME_TO_LOAD_TEST + i * TIME_TO_REACH_NEXT_PROMPT);
```

Comment:
Here is the dirty part... and the reason of the issues.
I think the inputs were send before the prompt requested them in some tests, so some random tests were waiting and timed out, or some nock call where never executed by the commands.

I increase the initial time to load from 500ms to 1000ms.
Open to better (quick) solutions.

Reply:
Ok, so if I get it right, you increased the timeout for the next (following) input ?
Seems to solve the issue at least for now (I launched the test 25 times from using a bash script just to be sure).

This is indeed not a definitive fix, but the original implementation already seems to go that way, and with the current test code structure I can't really think of a better solution